sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -27,13 +27,25 @@ public class ProcessMessageEventArgs : EventArgs
         /// </summary>
         public CancellationToken CancellationToken { get; }
 
+        /// <summary>
+        /// An event that is raised when the message lock is lost. This event is only raised for the scope of the Process Message handler.
+        /// Once the handler returns, the event will not be raised. There are two cases in which this event can be raised:
+        ///     1) When the message lock has expired based on the <see cref="ServiceBusReceivedMessage.LockedUntil"/> property.
```

Comment:
We should consider using an actual `<list>` element here so that it renders nice.